Learn HTML: character encoding (#8817)
GoogleChrome/web.dev: src/site/content/en/learn/html/document-structure/index.md

The document metadata, including the document title, character set, viewport settings, description, base URL, stylesheet links, and icons, are found in the `<head>` element. While you may not need all these features, always include character set, title, and viewport settings.

### Character encoding 

> The very first element in the `<head>` should be the character declaration. It comes before the title to ensure the browser can render the characters in that title and all the characters in the rest of the document.  

Author: ```suggestion
The very first element in the `<head>` should be the `charset` character encoding declaration. It comes before the title to ensure the browser can render the characters in that title and all the characters in the rest of the document.  
```

> The default language encoding in most browsers is `ISO-8859-1`, the Latin character set. However, you should use UTF-8, as it contains most of the characters you didn't even know existed. Also, it's the encoding type required by HTML5. 

Author: ```suggestion
The default encoding in most browsers is `ISO-8859-1`. However, you should use UTF-8, as it enables the two- to four-byte encoding of all characters, even ones you didn't even know existed. Also, it's the encoding type required by HTML5. 
```

> The default encoding in most browsers is `ISO-8859-1`. However, you should use UTF-8, as it enables the two- to four-byte encoding of all characters, even ones you didn't even know existed. Also, it's the encoding type required by HTML5. 

Author: ```suggestion
The default encoding in most browsers is `ISO-8859-1`. However, you should use [`UTF-8`](https://developer.mozilla.org/en-US/docs/Glossary/UTF-8), as it enables the two- to four-byte encoding of all characters, even ones you didn't even know existed. Also, it's the encoding type required by HTML5. 
```

> The default encoding in most browsers is `ISO-8859-1`. However, you should use [`UTF-8`](https://developer.mozilla.org/en-US/docs/Glossary/UTF-8), as it enables the two- to four-byte encoding of all characters, even ones you didn't even know existed. Also, it's the encoding type required by HTML5. 

Author: ```suggestion
The default encoding in most browsers is `windows-1252`. However, you should use [`UTF-8`](https://developer.mozilla.org/en-US/docs/Glossary/UTF-8), as it enables the two- to four-byte encoding of all characters, even ones you didn't even know existed. Also, it's the encoding type required by HTML5. 
```

> The default encoding in most browsers is `windows-1252`. However, you should use [`UTF-8`](https://developer.mozilla.org/en-US/docs/Glossary/UTF-8), as it enables the two- to four-byte encoding of all characters, even ones you didn't even know existed. Also, it's the encoding type required by HTML5. 

Author: ```suggestion
The [default encoding](https://html.spec.whatwg.org/multipage/parsing.html#documentEncoding) in most browsers is `windows-1252`, depending on the locale. However, you should use [`UTF-8`](https://developer.mozilla.org/en-US/docs/Glossary/UTF-8), as it enables the two- to four-byte encoding of all characters, even ones you didn't even know existed. Also, it's the encoding type required by HTML5. 
```

> The [default encoding](https://html.spec.whatwg.org/multipage/parsing.html#documentEncoding) in most browsers is `windows-1252`, depending on the locale. However, you should use [`UTF-8`](https://developer.mozilla.org/en-US/docs/Glossary/UTF-8), as it enables the two- to four-byte encoding of all characters, even ones you didn't even know existed. Also, it's the encoding type required by HTML5. 

Reviewer: ```suggestion
The [default encoding](https://html.spec.whatwg.org/multipage/parsing.html#documentEncoding) in most browsers is `windows-1252`, depending on the locale. However, you should use [`UTF-8`](https://developer.mozilla.org/docs/Glossary/UTF-8), as it enables the two- to four-byte encoding of all characters, even ones you didn't even know existed. Also, it's the encoding type required by HTML5. 
```
